doc-content/jbpm-docs/src/main/asciidoc/Persistence/auditlog-datamodel-ref.adoc

```diff
@@ -153,13 +153,17 @@ Whenever a node instance is entered from one of its incoming connections or is e
 |
 
 |``nodeContainerId``
-|The identifier of the container, if the node is inside an embedded sub-process node 
+|The identifier of the container, if the node is inside an embedded sub-process node
 |
 
 |``referenceId``
 |The reference identifier
 |
 
+|``observation``
+|The original node instance ID and job ID. You can use this information to trigger the job again.
```

Comment:
not always this is specify to the type of event. In the case the node instace log is related to scheduling event. then this is correct.